src/briefcase/integrations/android_sdk.py

```diff
@@ -377,7 +377,7 @@ def select_target_device(self, device_or_avd):
             if avd:
                 # It's a running emulator
                 running_avds[avd] = d
-                full_name = "@{avd} ({name} emulator)".format(
+                full_name = "emulator (@{avd} {name})".format(
```

Comment:
Cosmetically, I think the old construction for emulators read better. The content in brackets should be the qualifier, not the "meat" of the description.

Reply:
OK! Either way works for me, honestly.